test/e2e/config/azure-dev.yaml

```diff
@@ -97,7 +97,7 @@ providers:
 
 variables:
   KUBERNETES_VERSION: "${KUBERNETES_VERSION:-v1.22.1}"
-  AKS_KUBERNETES_VERSION: "${KUBERNETES_VERSION:-v1.19.9}"
+  AKS_KUBERNETES_VERSION: "${KUBERNETES_VERSION:-v1.22.6}"
```

Comment:
it says 1.22.6 but the code actually uses the latest 1.22 version (we changed it to the "stable-" pattern later in 1.0.